packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -324,9 +324,15 @@ class TransitLegBody extends Component<Props, State> {
                   {showViewTripButton && (
                     <ViewTripButton
                       tripId={leg.tripId}
-                      fromIndex={leg.from.stopIndex}
+                      fromIndex={
+                        leg.from.stopIndex ||
+                        leg?.trip?.departureStoptime?.stopPosition
+                      }
                       setViewedTrip={setViewedTrip}
-                      toIndex={leg.to.stopIndex}
+                      toIndex={
+                        leg.to.stopIndex ||
```

Comment:
are we keeping this old `.to.stopIndex` on here for legacy support? I guess it doesn't hurt anyone. But I think the newer one should be prioritized.

Reply:
Oh yes this is in here for now since itinerary-body still supports OTP1